o_el_iks.API/Program.cs

```diff
@@ -38,16 +38,31 @@
     .WithName("GetWeatherForecast")
     .WithOpenApi();
 
+List<RegistrationData> users = new List<RegistrationData>();
 
-app.MapPost("/register", ( RegistrationData data) =>
+app.MapPost("/register", ([FromBody] RegistrationData data) =>
 {
     if (string.IsNullOrWhiteSpace(data.email) || string.IsNullOrWhiteSpace(data.password))
     {
         return Results.BadRequest("Incorrect data.");
     }
+    RegistrationData newUser = new RegistrationData { email = data.email, password = data.password };
+    users.Add(newUser);
     return Results.Ok("Registration successful.");
 });
 
+app.MapPost("/logging", ([FromBody] LoggingData data) =>
```

Comment:
nazwa urla do zmiana, logging w znaczeniu programowania ma zupełnie inne znaczenie niż logowanie. Raczej zmieniłbym na sign-in